graylog2-web-interface/src/components/graylog/DropdownButton.jsx

```diff
@@ -1,24 +1,15 @@
-import React, { forwardRef, useCallback } from 'react';
+import React from 'react';
 // eslint-disable-next-line no-restricted-imports
 import { DropdownButton as BootstrapDropdownButton } from 'react-bootstrap';
 import styled from 'styled-components';
 
+import menuItemStyles from './styles/menuItem';
 import buttonStyles from './styles/button';
-import { propTypes, defaultProps } from './props/button';
 
-const DropdownButton = forwardRef((props, ref) => {
-  const { bsStyle, active, disabled } = props;
-  const StyledDropdownButton = useCallback(styled(BootstrapDropdownButton)`
-    ${buttonStyles(props)};
-  `, [bsStyle, active, disabled]);
+const DropdownButton = React.memo(styled(BootstrapDropdownButton)`
```

Comment:
You are using React.memo here to avoid unnecessary rendering of the Bootstrap component, right? Out of interest, how do you decide, when to use React.memo?

Reply:
It was added in concern of too having to generate styles and slowing down the render.
Removed it until we notice places of concern